### PR TITLE
chore: set max SDK version to <3.0.0

### PR DESCRIPTION
**Pull request by chalin**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -19,4 +20,4 @@ dev_dependencies:
   built_value_generator: ^5.1.0
```
Comment (kevmoo) [inline pubspec.yaml]: Bump `build_runner: ^0.9.2`

File: pubspec.yaml [hunk 2]
```diff
@@ -19,4 +20,4 @@ dev_dependencies:
...
   dart_style: ^1.0.0
   source_gen: '>=0.7.5 <0.9.0'
-  test: ^0.12.0
+  test: '>=0.12.0 <2.0.0'
```
Comment (kevmoo) [inline pubspec.yaml]: Might as well make this `^1.3.0`

File: pubspec.yaml (hunk 1 above)
Comment (chalin) [reply pubspec.yaml]: done

File: pubspec.yaml (hunk 2 above)
Comment (chalin) [reply pubspec.yaml]: done

**Pull request merged**